src/pages-and-resources/discussions/app-config-form/apps/shared/DivisionByGroupFields.jsx

```diff
@@ -14,10 +14,11 @@ function DivisionByGroupFields({
 }) {
   return (
     <>
-      <h5>{intl.formatMessage(messages.divisionByGroup)}</h5>
+      <h5 className="text-gray-500 mt-4">{intl.formatMessage(messages.divisionByGroup)}</h5>
```

Comment:
Nit: in general, it's a good practice to try to consistently use bottom margins to space things out, rather than top margins.  This way each component is responsible for the space it wants below it.  It makes things a bit more predictable.  In this component for instance, I'd have this `h5` define `mb-3` to space it out from the FormSwitchGroup below.  The `mt-4` here is something that I'd probably add to the parent component/previous component in the parent.